### PR TITLE
Added JS3 week1 homework

### PR DESCRIPTION
**Pull request by ajgcph2019**

### CODE REVIEW

File: javascript/javascript3/week1/weather/weather.js [hunk 1]
```diff
@@ -0,0 +1,99 @@
+const name = document.querySelector("#city");
```
Comment (nmoskaleva) [inline javascript/javascript3/week1/weather/weather.js:1]: `name` is not a good name:) `countryName` is a good one

File: javascript/javascript3/week1/weather/weather.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  function error() {
+    console.log("error");
+  }
+});
+
+function weatherInformation() {
+  const url =
+    "https://api.openweathermap.org/data/2.5/weather?q=" +
+    input.value +
+    "&appid=bcd019f0d80640e0e14d9273d6f227d0&units=metric";
```
Comment (nmoskaleva) [inline javascript/javascript3/week1/weather/weather.js:39]: on line 73 you use string literals, that is nicer

File: javascript/javascript3/week1/weather/weather.js [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+const windSpeed = document.querySelector(".wind");
+const humidity = document.querySelector(".humidity");
+const sunRise = document.querySelector(".sunRise");
+const sunSet = document.querySelector(".sunSet");
+const displayCity = document.querySelector(".btn");
+
+const input = document.querySelector("#searchInput");
+const currentLocBtn = document.querySelector("#location");
+
+displayCity.addEventListener("click", () => {
```
Comment (nmoskaleva) [inline javascript/javascript3/week1/weather/weather.js:15]: `displayCity` is a bit confusing, it's not clear that it's a button

File: javascript/javascript3/week1/weather/weather.js [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+const temperature = document.querySelector("#temp");
+const icon = document.querySelector("#icon");
+const desc = document.querySelector(".desc");
+const windSpeed = document.querySelector(".wind");
+const humidity = document.querySelector(".humidity");
+const sunRise = document.querySelector(".sunRise");
+const sunSet = document.querySelector(".sunSet");
+const displayCity = document.querySelector(".btn");
+
+const input = document.querySelector("#searchInput");
```
Comment (nmoskaleva) [inline javascript/javascript3/week1/weather/weather.js:12]: CSS classes and ids should be in kebab-case: `searchInput` => `search-input`

File: javascript/javascript3/week1/createOwnJson/createJsonFile.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
+const myFavouritesJsObject = {
+    "books " : ["Famous Five", "Harry Potter" , "Kite Runner"] ,
```
[the remark this answers is not in the document]
Comment (ajgcph2019) [reply javascript/javascript3/week1/createOwnJson/createJsonFile.js:2]: Corrected it 
const myFavouritesJsObject = {
    books  : ["Famous Five", "Harry Potter" , "Kite Runner"] ,
    series : ["Friends" ,"Suits","Game Of Thrones"] ,
    authors : ["Enid Blyton","JK Rowling"]

};

const toJson = JSON.stringify(myFavouritesJsObject);
document.getElementById("demo1").innerHTML = toJson;

const jsonObject = `
  {  "books" : ["Famous Five", "Harry Potter" , "Kite Runner"] ,
    "series" : ["Friends" ,"Suits","Game Of Thrones"] ,
    "authors" : ["Enid Blyton","JK Rowling"] 

 }`;
 
 const toJsObject1 = JSON.parse(jsonObject);
 document.getElementById("demo2").innerHTML = `${toJsObject1.authors},${toJsObject1.series}` ;

File: javascript/javascript3/week1/weather/weather.js (hunk 4 above)
Comment (ajgcph2019) [reply javascript/javascript3/week1/weather/weather.js:12]: thank you for telling. I have made the change

**Pull request merged**